docker/0.2.2/full-tls-mainnet/docker-compose.yaml

```diff
@@ -0,0 +1,123 @@
+
+  fedimintd:
+    image: fedimint/fedimintd:v0.2.2
+    volumes:
+      - fedimintd_data:/data
+    ports:
+      - "0.0.0.0:8173:8173"
+    environment:
+      # Starts out using esplora but should move to dedicated bitcoin core
+      - FM_BITCOIN_RPC_KIND=esplora
+      - FM_BITCOIN_RPC_URL=https://mutiny.mempool.space/api/
+      - FM_BITCOIN_NETWORK=bitcoin
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_P2P_URL=fedimint://fedimint.my-super-host.com:8173
+      - FM_BIND_API=0.0.0.0:80
+      - FM_API_URL=wss://fedimintd.fedimint.my-super-host.com
+    restart: always
+    platform: linux/amd64
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.fedimintd.loadbalancer.server.port=80"
+      - "traefik.http.routers.fedimintd.rule=Host(`fedimintd.fedimint.my-super-host.com`)"
+      - "traefik.http.routers.fedimintd.entrypoints=websecure"
+      - "traefik.http.routers.fedimintd.tls.certresolver=myresolver"
+
+  guardian-ui:
+    image: fedimintui/guardian-ui:0.2.1
+    environment:
+      - PORT=80
+      - REACT_APP_FM_CONFIG_API=wss://fedimintd.fedimint.my-super-host.com
+    depends_on:
+      - fedimintd
+    restart: always
+    platform: linux/amd64
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.guardian-ui.loadbalancer.server.port=80"
+      - "traefik.http.routers.guardian-ui.rule=Host(`guardian-ui.fedimint.my-super-host.com`)"
+      - "traefik.http.routers.guardian-ui.entrypoints=websecure"
+      - "traefik.http.routers.guardian-ui.tls.certresolver=myresolver"
+
+  ### START_OF_GATEWAY ###
+  gatewayd:
+    image: fedimint/gatewayd:v0.2.2
+    command: gatewayd lnd
+    environment:
+      # Path to folder containing gateway config and data files
+      - FM_GATEWAY_DATA_DIR=/gateway_data
+      # Gateway webserver listen address
+      - FM_GATEWAY_LISTEN_ADDR=0.0.0.0:80
+      # Public URL from which the webserver API is reachable
+      - FM_GATEWAY_API_ADDR=https://gatewayd.fedimint.my-super-host.com
+      # Gateway webserver authentication password
+      - FM_GATEWAY_PASSWORD=thereisnosecondbest
+      # Configured gateway routing fees Format: <base_msat>,<proportional_millionths>
+      - FM_GATEWAY_FEES=1000,10000
+      # LND RPC address
+      - FM_LND_RPC_ADDR=lnd_gprc_url
+      # LND TLS cert file path
+      - FM_LND_TLS_CERT=/lnd_data/tls.cert
+      # LND macaroon file path
+      - FM_LND_MACAROON=/lnd_data/admin.macaroon
+    volumes:
+      - gateway_datadir:/gateway_data
+      - ~/.lnd:/lnd_data:ro
+    restart: always
+    platform: linux/amd64
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.gatewayd.loadbalancer.server.port=80"
+      - "traefik.http.routers.gatewayd.rule=Host(`gatewayd.fedimint.my-super-host.com`)"
+      - "traefik.http.routers.gatewayd.entrypoints=websecure"
+      - "traefik.http.routers.gatewayd.tls.certresolver=myresolver"
+
+  gateway-ui:
+    image: fedimintui/gateway-ui:0.2.1
```

Comment:
v0.2.2 as well? Or was there an auth issue with that?

Reply:
same as prev comment